onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -528,14 +528,6 @@ Sample echo operator.)DOC");
       .TypeConstraint("T", {"tensor(float)"}, "Constrain input0 and output types to float tensors")
       .TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
         ONNX_NAMESPACE::propagateElemTypeFromInputToOutput(ctx, 0, 0);
-        if (ctx.getNumOutputs() > 1) {
-          // MaxPool with two outputs case.
-          auto output_type = ctx.getOutputType(1);
-          if (output_type->value_case() == ONNX_NAMESPACE::TypeProto::kTensorType ||
-              output_type->value_case() == ONNX_NAMESPACE::TypeProto::VALUE_NOT_SET) {
-            output_type->mutable_tensor_type()->set_elem_type(ONNX_NAMESPACE::TensorProto::INT64);
-          }
-        }
```

Comment:
just curious, was this added by onnx or by your last pr?

Reply:
ConvPoolTypeAndShapeInference was changed in onnx to ConvPoolShapeInference...
This type inference code existed before in ConvPoolTypeAndShapeInference... When making the changes in this file as part of the update onnx PR I should not have just copied it to this op since this op does not have the 2nd output.